aws/otel-agent-config.yaml

```diff
@@ -0,0 +1,23 @@
+extensions:
+  health_check:
+    endpoint: 0.0.0.0:13133
+
+receivers:
+  zipkin:
```

Comment:
Maybe add a comment that the pulumi code will set `const zipkinPort = 9411;`?

Reply:
Instead, why not use this file as the source of truth, and read/parse it in `index.ts`? It is just regular ts code, and we can add a YAML parser as a dependency.

Reply:
@marcospereira Let's not overcomplicate. 9411 is the default port and that is the only reason why on the TS code it's specified. This YAML file is inheriting a lot of other defaults.

We may add an extra comment with a link to the OTel docs for each of the converter/receivers/etc... but it's all vanilla OTel.

Reply:
> @marcospereira Let's not overcomplicate. 9411 is the default port and that is the only reason why on the TS code it's specified. This YAML file is inheriting a lot of other defaults.

I don't see how parsing the YAML will overcomplicating things here. From a user point of view, it seems even easier to reason about: you do the OTEL collector configuration the way it is suppose to be done (and hopefully documented by them), and that is it.

> We may add an extra comment with a link to the OTel docs for each of the converter/receivers/etc... but it's all vanilla OTel.

My point here was not about not following the defaults, but instead, if you want to change them for whichever reason, you now need to understand OTEL collector configuration **and** some very specific details of our template. Making the template read the file would solve that.

Reply:
I've implemented the zipkin and health check ports parsing with user-friendly error handling.